ext/hal/ti/simplelink/Kbuild

```diff
@@ -35,3 +36,10 @@ obj-$(CONFIG_SIMPLELINK_HOST_DRIVER) += kernel/zephyr/dpl/MutexP_zephyr.o
 obj-$(CONFIG_SIMPLELINK_HOST_DRIVER) += kernel/zephyr/dpl/SemaphoreP_zephyr.o
 obj-$(CONFIG_SIMPLELINK_HOST_DRIVER) += kernel/zephyr/dpl/ClockP_zephyr.o
 obj-$(CONFIG_SIMPLELINK_HOST_DRIVER) += kernel/zephyr/dpl/HwiP_zephyr.o
+endif
+
+ifeq ($(CONFIG_HAS_MSP432P4XXSDK),y)
```

Comment:
Perhaps, since now there is an ifeq guard, the ccflags- and obj-$(CONFIG_HAS...) can be replaced by ccflags-y and obj-y everywhere?